planner/core/cbo_test.go

```diff
@@ -278,7 +278,7 @@ func (s *testAnalyzeSuite) TestIndexRead(c *C) {
 		},
 		{
 			sql:  "select count(*) from t where e > 1 group by b",
-			best: "IndexLookUp(Index(t.b)[[NULL,+inf]], Table(t)->Sel([gt(test.t.e, 1)]))->StreamAgg",
+			best: "TableReader(Table(t)->Sel([gt(test.t.e, 1)])->HashAgg)->HashAgg",
```

Comment:
For reviewers: before this PR, column `b` is not analyzed because of the existence of index on `b`, so estimated NDV of b is about 8000; after this PR, estimated NDV of `b` is 100 which is consistent with data, and the plan is changed due to this NDV change.